### PR TITLE
Ampplitude: addressing CFD for revenue field , when the value is 0

### PR DESCRIPTION
**Pull request by shrouti1507**
## Description of the change
[Click here](https://rudderlabs.slack.com/archives/C02A3UAU9UN/p1638508147258000)

### CODE REVIEW

**Review by ruchiramoitra — COMMENTED** [review]
LGTM

**Comment by ruchiramoitra** [comment]
@shrouti1507  Can we also check line no. 503 with the same logic? please check once

File: v0/destinations/am/transform.js [hunk 1]
```diff
@@ -622,7 +623,8 @@ function process(event) {
   const messageType = message.type.toLowerCase();
   const toSendEvents = [];
   if (messageType === EventType.TRACK) {
-    if (message.properties && message.properties.revenue) {
+    const { properties } = message;
+    if (properties && isDefinedAndNotNull(properties.revenue)) {
```
Comment (saikumarrs) [inline v0/destinations/am/transform.js:627]: Please add new test cases to cover this bug scenario

**Review by saikumarrs — CHANGES_REQUESTED** [review]
LGTM

File: v0/destinations/am/transform.js (hunk 1 above)
Comment (shrouti1507) [reply v0/destinations/am/transform.js:627]: Added test case

**Review by saikumarrs — APPROVED** [review]

**Review by ruchiramoitra — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**